css/style.css

```diff
@@ -0,0 +1,799 @@
+  color: rgb(236, 44, 11);
+  font-size: 13px;
+  text-transform: uppercase;
+  width: 60%;
+  margin-bottom: 10px;
+}
+
+.section-details .second-latest-article .img-side {
+  width: 90px;
+  height: 100%;
+}
+
+/* END all sections */
+
+/* Latest Deals */
+
+.post-img-deals {
+  display: block;
+  height: 135px;
+  max-width: 100%;
+  margin-bottom: 8px;
+  background-size: cover;
+  background-image: url(https://cdnp1.stackassets.com/6c14a9f8eb97934746690af56dfa58f4fe0f4de7/store/21f45a858097668f3c5c0419808ec1f79bb6c68b09295123ebe76d0730e2/sale_24386_primary_image.jpg);
+}
+
+/* END Latest Deals */
+
+/* Footer */
+.footer {
+  background: #131313;
+  color: #aaa;
+}
+
+.footer_links {
+  padding: 42px 0 28px 0;
+}
+
+.footer_links_social-media {
+  height: 45px;
+  margin-bottom: 21px;
+}
+
+.footer_links--inline {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer_links_social-media li i {
+  margin: 10px;
+  color: white;
+}
+
+.footer_wrapper--column {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer_copyright_logo {
+  height: 9px;
+  margin: 0 1px;
+}
+
+.footer_copyright {
+  background: #000;
+  padding: 32px 0;
+  line-height: 1.5em;
+}
+
+.footer_copyright span {
+  margin: 0 2.5px;
+}
+
+.footer_copyright b {
+  color: red;
+}
+
+/* END Footer */
+
+@media only screen and (max-width: 1024px) {
+  .logo {
+    display: flex;
+    z-index: 999;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .top-nav,
+  .main-nav {
+    margin-left: 106px;
+  }
+
+  .article-big {
+    background:
+      linear-gradient(
+        180deg,
+        rgba(2, 0, 36, 1) 0%,
+        rgba(9, 9, 121, 0) 0%,
+        rgba(0, 0, 0, 1) 100%
+      ),
+      url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/12/1-13-796x417.jpg)
+        center no-repeat;
+    background-size: cover;
+  }
+
+  .article-big-logo {
+    visibility: hidden;
+  }
+
+  .list-articles {
+    justify-content: flex-start;
+  }
+
+  .list-articles .latest-li {
+    flex: 0 1 30.3%;
+    margin: 10px;
+  }
+
+  .funding-li li {
+    flex: 0 1 30.33%;
+    margin: 1%;
+    margin-top: -80px;
+  }
+
+  #visnone {
+    display: none;
+  }
+
+  .section-details {
+    flex: 0 1 50%;
+    max-width: 100%;
+    padding: 8px 20px;
+    margin: 0;
+  }
+
+  .list-articles .deals-li {
+    flex: 0 1 30.33%;
+    margin: 10px;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  nav {
+    height: 48px;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .logo {
+    height: 48px;
+  }
+
+  .cover {
+    display: block;
+  }
+
+  .article-big {
+    height: 239.1px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .article-big-logo {
+    display: none;
+  }
+
+  .article-big .cover-article-info {
+    margin: 20px;
+  }
+
+  .article-big .article-section {
+    color: white;
+    font-size: 0.87 em;
```

Comment:
Line number 731 : invalid property value
Kindly recommended to delete space (write 0.87em instead 0.87 em)